ucm2/sof-soundwire/HiFi.conf

```diff
@@ -2,9 +2,197 @@
+		]
+	}
+}
+
+If.seq2p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA2.0 2 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA2.0 2 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq3p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA3.0 3 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA3.0 3 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq4p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA4.0 4 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA4.0 4 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq5p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA5.0 5 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA5.0 5 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq6p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA6.0 6 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA6.0 6 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq7p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA7.0 7 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA7.0 7 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq8p {
+	Condition {
+		Type ControlExists
+		Control "name='PGA8.0 8 Master Playback Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA8.0 8 Master Playback Volume' 32"
+		]
+	}
+}
+
+If.seq1c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA1.0 1 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA1.0 1 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq2c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA2.0 2 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA2.0 2 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq3c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA3.0 3 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA3.0 3 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq4c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA4.0 4 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA4.0 4 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq5c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA5.0 5 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA5.0 5 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq6c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA6.0 6 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA6.0 6 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq7c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA7.0 7 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA7.0 7 Master Capture Volume' 50"
+		]
+	}
+}
+
+If.seq8c {
+	Condition {
+		Type ControlExists
+		Control "name='PGA8.0 8 Master Capture Volume'"
+	}
+	True {
+		EnableSequence [
+			cset "name='PGA8.0 8 Master Capture Volume' 50"
+		]
+	}
+}
```

Comment:
This is not correct. If multiple controls can be present.

Use `EnableSequence.8.cset "name='PGA8.0 8 Master Capture Volume' 50"` for example.

But the default volumes should not be set for UCM2. Handle the volume settings in the device section (using PlaybackMixerElem and CaptureMixerElem and PlaybackMasterElem and  CaptureMasterElem).

Reply:
Hi Jaroslav,

Sorry for the delay reply. I didn't get the email notification and I missed you comments. I tried your suggestion of "EnableSequence.8.cset" and it seems not to work. I will describe more details in my patch email thread, so we can talk the problem in one place.

I move the volume settings in HiFi, this is because we are designing a general UCM2 solution. So on some platforms, for example, “PGA3.0 3” kcontrol is used in RT1308, but maybe on some platform, RT1308 may not use "PGA3.0 3" kcontrol. This is why I don't want to set the default volumes in the device section.